### DISCUSSION TITLE
tpws socks statistic and status

### DISCUSSION THREAD

**Discussion by pimlab22** [bol-van/zapret#823] [Ideas]
Возможно ли сделать , чтобы tpws в режиме socks на запрос "ip:1080/stat" отвечал статистикой работы? Таким образом можно будет с помощью "monitrc" отследить не завис ли процесс, а с помощью "munin" собрать статистику работы - чертить графики.

**Comment by bol-van** [comment]
tpws дает сокс прокси, а не http
теоретически, конечно, сделать можно мини веб сервер. но делать это не планируется, тк работа не на раз два
слишком мало кому надо

процесс зависать не должен. если он завис, значит там что-то не так работает.
и в крайнем случае можно через него делать curl куда-нибудь (только не на сам хост, где стоит tpws, это не дадут)

**Reply by pimlab22** [reply]
Жаль конечно, но стоило спросить
